deploy-board/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
+ENV PYTHONUNBUFFERED 1 
+
+RUN mkdir /opt/deploy-board
+WORKDIR /opt/deploy-board
```

Comment:
We can refactor the /opt/deploy-board to be referred via a single 
ENV PROJECT_DIR = /opt/deploy-board

Then say
RUN mkdir $PROJECT_DIR